src/main/java/com/puppycrawl/tools/checkstyle/Checker.java

```diff
@@ -190,7 +188,8 @@ public void destroy() {
                 cacheFile.persist();
             }
             catch (IOException ex) {
-                throw new IllegalStateException("Unable to persist cache file.", ex);
+                final String destroyMessage = getLocalizedMessage("Checker.destroy");
```

Comment:
Why do we need it as a separate variable? Can we use it inplace `throw new IllegalStateException(getLocalizedMessage("Checker.destroy"), ex);`?

Reply:
Done..
Even more optimization, that's fantastic